src/app.js

```diff
@@ -58,13 +60,13 @@ app.use('/execute', async (req, res) => {
   const reqStep = new Step(token, pattern, table)
   const stepDef = testContext.getMatch(reqStep)
 
-  console.log('Executing new step:\n', reqStep)
+  log.info('Executing new step: ' + JSON.stringify(reqStep))
```

Comment:
```suggestion
  log.info('Executing new step:\n ' + JSON.stringify(reqStep))
```
newline would be nice here

Reply:
not a good idea IMO to have line breaks in logs.